### PR TITLE
Adds read-me file

### PR DESCRIPTION
**Pull request by OussEmaDevCode**
This pull request adds a descriptive and consistent read me to the ppi-vision repository.

### CODE REVIEW

**Comment by OussEmaDevCode** [comment]
@apoorva-mk @Nayan Hey, please have a look at this !

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<img height='200' src="https://user-images.githubusercontent.com/37406965/51083189-d5dc3a80-173b-11e9-8ca0-28015e0893ac.png" align="left" hspace="1" vspace="1">
+
+# PPI-Vision Android Application for MifosX
+
+Vision PPI is a computer vision and machine learning based app to help in filling out the PPI Survey. There are two broad aspects to this project - The android app which provides the interface to the field officer for conducting the survey and the machine learning models which will be used to analyze the images captured.
+## Screenshots
+
+<p>
+  <img src="https://user-images.githubusercontent.com/38397893/72104939-c1ba2180-332c-11ea-8671-c8191c3927ec.png" />
```
Comment (yashk2000) [inline README.md:9]: Combining all 4 screenshots into one single image makes it look a bit congested. Could you please make a 2*2 table and place the 4 images separately on it. That ought to make the readme look neater.

**Comment by OussEmaDevCode** [comment]
I am a little busy in the moment I'll do that the moment I get some free time

**Comment by OussEmaDevCode** [comment]
@yashk2000 Done !!!!

**Comment by yashk2000** [comment]
@OussEmaDevCode great work!
Thanks for the contribution :rocket:

**Pull request merged**